app/components/Button/index.tsx

```diff
@@ -35,8 +34,8 @@ const Button = ({
     .join(' ')
   if (typeof action === 'string') {
     return (
-      <Link
-        to={action}
+      <a
+        href={action}
         className={classes}
```

Comment:
@mruwnik I replaced the <Link> with <a>, because the link element didn't seem to be functioning. Is that ok?

Reply:
It would be better as a `Link`, for efficiency reasons. `a` will cause a full fetch of the linked page, with a bunch of stuff getting fetched. `Link` only does that for external references - going to local pages happens via js, not the browser. This can be a significant speed up, especially when someone has slow/expensive internet

Reply:
That being said, the `Link` element has a problem when you want to navigate to the same page that you're on. This I don't have a good solution for. You could go in all hacky, and instead of providing `action="/some/url"`, do it as `action={() => { window.location.href = "/some/url"}}`